jsapp/js/dataInterface.d.ts

```diff
@@ -242,3 +262,51 @@ interface AssetResponse {
   settings__form_id?: string
   settings__title?: string
 }
+
+interface PaginatedResponse {
+  count: number
+  next: null | string
+  previous: null | string
+  results: any[]
```

Comment:
Curious if a `Submission` type is a todo, or there's another reason why this is `any`?